docs/Using-Fleet/FAQ.md

```diff
@@ -1,35 +1,53 @@
-- [How do I revoke the authorization tokens for a user?](#how-do-i-revoke-the-authorization-tokens-for-a-user)
-- [How do I monitor the performance of my queries?](#how-do-i-monitor-the-performance-of-my-queries)
-- [How do I monitor a Fleet server?](#how-do-i-monitor-a-fleet-server)
-- [Can I disable password-based authentication in the Fleet UI?](#can-i-disable-password-based-authentication-in-the-fleet-ui)
-- [Where are my query results?](#where-are-my-query-results)
-- [Why aren’t my live queries being logged?](#why-arent-my-live-queries-being-logged)
-- [Can I use the Fleet API to fetch results from a scheduled query pack?](#can-i-use-the-fleet-api-to-fetch-results-from-a-scheduled-query-pack)
-- [How do I automatically add hosts to packs when the hosts enroll to Fleet?](#how-do-i-automatically-add-hosts-to-packs-when-the-hosts-enroll-to-Fleet)
-- [How do I automatically assign a host to a team when it enrolls with Fleet?](#how-do-i-automatically-assign-a-host-to-a-team-when-it-enrolls-with-fleet)
-- [How do I resolve an "unknown column" error when upgrading Fleet?](#how-do-i-resolve-an-unknown-column-error-when-upgrading-fleet)
-- [Why my host is not updating a policy's response.](#why-my-host-is-not-updating-a-policys-response)
-- [What should I do if my computer is showing up as an offline host?](#what-should-i-do-if-my-computer-is-showing-up-as-an-offline-host)
-- [How does Fleet deal with IP duplication?](#how-does-fleet-deal-with-ip-duplication)
-- [Can Orbit run alongside osquery?](#can-orbit-run-alongside-osquery)
-- [Can I control how Orbit handles updates?](#can-i-control-how-orbit-handles-updates)
-- -[When will the newest version of osquery be available to Orbit?](#when-will-the-newest-version-of-osquery-be-available-to-orbit)
-- [Can I bundle osquery extensions into Orbit?](#can-i-bundle-osquery-extensions-into-orbit)
-- [How does Fleet work with osquery extensions?](#how-does-fleet-work-with-osquery-extensions)
-- [Why am I seeing "unknown certificate error" when adding hosts to my dev server?](#why-am-i-seeing-"unknown-certificate-error"-when-adding-hosts-to-my-dev-server)
-- [Can I hide known vulnerabilities that I feel are insignificant?](#can-i-hide-known-vulnerabilities-that-i-feel-are-insignificant)
-- [Can I create reports based on historical data in Fleet?](#can-i-create-reports-based-on-historical-data-in-fleet)
-- [Why can't I run queries with `fleetctl` using a new API-only user?](#why-cant-i-run-queries-with-fleetctl-using-a-new-api-only-user)
-- [Can I audit actions taken in Fleet?](#can-i-audit-actions-taken-in-fleet)
-- [How often is the software inventory updated?](#how-often-is-the-software-inventory-updated)
-- [Can I group results from multiple hosts?](#can-i-group-results-from-multiple-hosts)
-- [How do I downgrade from Fleet Premium to Fleet Free?](how-do-i-downgrade-from-fleet-premium-to-fleet-free)
+- [Using Fleet FAQ](#using-fleet-faq)
```

Comment:
This and the sub-items in e.g. `[Where are my query results?]` are automatically formatted by my markdown editor. (They attempt to generate the table of contents automatically.)

Let me know if I should revert this.